planet/cli/data.py

```diff
@@ -14,14 +14,14 @@
 # possible.
 
 
-def parse_item_types(ctx, param, value) -> list:
+def parse_item_types(ctx, param, value: str) -> List[str]:
     """Turn a string of comma-separated names into a list of names."""
     # Note: we could also normalize case and validate the names against
     # our schema here.
-    return [part.strip() for part in (value or "").split(",")]
+    return [part.strip() for part in value.split(",")]
```

Comment:
This is the way to go. Our function has no special cases (`if` or `or`) that aren't needed. The input `value` will always be a string unless we change the argument this callback is associated with.